packages/plugins/rubygem-foreman_remote_execution/rubygem-foreman_remote_execution.spec

```diff
@@ -129,11 +128,15 @@ install -Dp -m0644 %{buildroot}%{gem_instdir}/extra/cockpit/settings.yml.example
 %config(noreplace) %{_sysconfdir}/foreman/cockpit/cockpit.conf
 %config(noreplace) %{_sysconfdir}/foreman/cockpit/foreman-cockpit-session.yml
 %{_unitdir}/foreman-cockpit.service
+%{gem_instdir}/extra/cockpit
```

Comment:
Why do we need the extra directory? All files are copied, except `foreman-cockpit-session` which is a symlink. Perhaps we should simply make line 84 an `install` and `%exclude %{gem_instdir}/extra` instead.

Reply:
maybe. I was not planing to re-think the packaging. but I can :)

Reply:
well, let's see how that goes